src/Tools/dotnet-counters/CounterMonitor.cs

```diff
@@ -357,7 +311,7 @@ private async Task<int> Start(bool liveMonitor)
             }
             if (!terminated)
             {
-                StopMonitor(liveMonitor);
+                renderer.Stop();
```

Comment:
Is there a path that gets here without already having called renderer.Stop()? It looked like we would be coming from the break on line 301 in which case StopMonitor() at 300 has already made this call.